marklogic-client-api/src/main/java/com/marklogic/client/impl/OkHttpServices.java

```diff
@@ -6270,4 +6292,28 @@ protected void set() {
         is = true;
     }
   }
+  
+  public class ConnectionResult {
```

Comment:
Would the code be easier to read and understand with correct indentation of the private fields and the first method?